src/styles/variables.css

```diff
@@ -0,0 +1,21 @@
+:root {
+  --color-heliow: #febb01;
```

Comment:
There's a lot of color around here. Is this minimal?

Reply:
We can remove to